src/components/Disputes/DisputeInfo.js

```diff
@@ -256,18 +249,41 @@ function Field({ label, value }) {
       >
         {label}
       </h2>
-      {isAddress(value) ? (
-        <div
-          css={`
-            display: flex;
-            align-items: flex-start;
-          `}
-        >
-          <LocalIdentityBadge
-            connectedAccount={addressesEqual(value, wallet.account)}
-            entity={value}
-          />
-        </div>
+
+      {typeof value === 'string' ? (
+        value.split('\n').map((line, i) => (
```

Comment:
Perhaps we can move this bit all into its own component, something like `DisputeTextField`